src/Aspire.Dashboard/Model/ResourceGraph/ResourceGraphMapper.cs

```diff
@@ -77,9 +77,9 @@ public static ResourceDto MapResource(ResourceViewModel r, IDictionary<string, R
         return dto;
     }
 
-    private static string ResolvedEndpointText(DisplayedEndpoint? endpoint)
+    private static string ResolvedEndpointText(DisplayedUrl? endpoint)
     {
-        var text = endpoint?.Text ?? endpoint?.Url;
+        var text = endpoint?.Url;
         if (string.IsNullOrEmpty(text))
         {
             return "No endpoints";
```

Comment:
This should say "No URLs". And while you're here, could you move the text to a resource file. I should have done that 😄

Reply:
Actually the resource graph only looks at endpoint-associated URLs right now. When it was showing all URLs it would pick the first one and then show the display text un-hyperlinked which wasn't useful at all. Having it show the actual endpoint address is more useful IMO.